src/Steps/TOInfo.tsx

```diff
@@ -0,0 +1,131 @@
+import { useContext } from "react";
+import { Col, Form, Row } from "react-bootstrap";
+import { globalContext } from '../stateManagement/GlobalStore';
+
+export const TOInfo = () => {
+  const { globalState, dispatch } = useContext(globalContext);
+  return (
+    <div className="m-3">
+      <h1>TMCR Information</h1>
```

Comment:
@RobertPorterfield - Do they have any backlog items to further refine the data that is input here, such as suggested placeholder text, required format/length/etc?

Reply:
Not yet, but the product owners do have an open task to look at these fields and provide me with feedback regarding any restrictions/placeholders/tooltips/etc. that they think their people will find useful. In the next meeting I'll ask them to identify some anticipated users that'll be able to do some testing and provide useful feedback to us.